docs/user-guide/images.md

```diff
@@ -100,6 +101,23 @@ Troubleshooting:
   - `plugins.go:56] Registering credential provider: aws-ecr-key`
   - `provider.go:91] Refreshing cache for provider: *aws_credentials.ecrProvider`
 
+### Using Azure Container Registry (ACR)
+When using Azure container registry you can authenticate using either an admin user or a service principal.
+In either case, authentication is done via standard Docker authentication.  These instructions assume the
+python based [az](https://github.com/azure/azure-cli) tooling.
```

Comment:
"The following instructions assume you're using the `azure-cli` command-line tool."